codespell_lib/data/dictionary.txt

```diff
@@ -1195,32 +1316,40 @@ attendents->attendants
 attened->attended
 attension->attention
 attept->attempt
+attetntion->attention
 attibute->attribute
 attibuted->attributed
 attibutes->attributes
 attirbute->attribute
 attirbutes->attributes
 attitide->attitude
+attmept->attempt
 attrbiute->attribute
 attribbute->attribute
 attribiute->attribute
 attribiutes->attributes
 attribtes->attribute
+attribtue->attribute
+attribude->attribute
+attribue->attribute
 attribues->attributes
 attribuite->attribute
 attribures->attributes
 attribut->attribute
 attributei->attribute
+attributen->attriburen
```

Comment:
should be `attribute`
note: thi may throw false positives for any german spelling